kubeflow/tf-serving/tf-serving.libsonnet

```diff
@@ -23,6 +23,20 @@ local networkSpec = networkPolicy.mixin.spec;
         },
       },
 
+      modelSecret(name, namespace, awsAccessKeyID, awsSecretAccessKey, labels={ app: name}): {
```

Comment:
I don't think we should create secrets in our ksonnet configs. The expectation of a secret is that its sensitive data.
So I'm assuming awsSecretAccessKey is a token/password that shouldn't be widely shared.

But the expectation is that deployment/ksonnet configs can be checked into source control.

If the data isn't sensitive why is it a secret and not a config map?